style.css

```diff
@@ -0,0 +1,371 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap");
+@import url("../fonts/cocogoose.css");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Lato", sans-serif;
+  width: 100%;
+  animation: fadeInAnimation ease 2s;
+  animation-iteration-count: 1;
+  animation-fill-mode: forwards;
+}
+
+@keyframes fadeInAnimation {
+  0% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+}
+
+header {
+  position: fixed;
+  width: 100%;
+  z-index: 10;
+}
+
+.logo img {
+  width: 100px;
+}
+
+nav ul .nav-item:last-child {
+  color: #ec5242;
+  border: 5px solid #ec5242;
+}
+
+nav ul li a.nav-link {
+  color: #272a31;
+}
+
+nav ul li a.nav-link:hover,
+nav ul li a.nav-link:active {
+  color: #ec5242 !important;
+}
+
+nav .nav-top {
+  background-color: #444 !important;
+  font-size: 14px !important;
+}
+
+nav {
+  background: white !important;
+  box-shadow: 0 0.125rem 0.25rem rgba(0, 0, 0, 0.075);
+}
+
+.hero {
+  background-color: #d3d3d3;
```

Comment:
The pages use the exact same colors, for text, backgrounds, borders, etc, from the original design 🔥 Well done!